_data/wai-evaluation-tools-list/submissions/2a6de8d0-3fd7-11f0-bca1-1b53378de832.json

```
@@ -0,0 +1,42 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "automated": [
+    "Single page/screen"
+  ],
+  "comments": "",
+  "contact": "apriore@tpgi.com",
+  "features": "Use TPGi's free color contrast checker to optimize your content - including text and visual elements - for individuals with color-blindness or low vision impairments.",
+  "form_name": "submission",
+  "form_version": "1",
+  "guideline": [
+    "WCAG 2.2",
+    "WCAG 2.1"
+  ],
+  "language": [
+    "en"
+  ],
+  "license": [
+    "Free",
+    ""
+  ],
+  "product": [
+    "Website"
+  ],
+  "provider": "TPGi",
+  "publish-permission": "on",
+  "purpose": [
+    "Automated testing",
+    "Manual testing"
```

```suggestion
    "Manual testing"
```
There is no evidence that the tool can perform automated testing.